BeatSpeedrun/Source/Managers/LevelPlayManager.cs

```diff
@@ -106,7 +107,7 @@ private void OnSongPaused()
                     return;
                 }
 
-                var defaultEnvironment = _customLevelLoader.LoadEnvironmentInfo(null, false);
```

Comment:
`CustomLevelLoader#LoadEnvironmentInfo`'s accessibility has been changed to private